### ISSUE TITLE
certmanager.installCRDs=true breaks installation

### ISSUE THREAD

**Issue by AntonAleksandrov13** [arttor/helmify#145]
Hi, 
I have been working with creating a Helm Chart for an internal operator. It requires cert-manager for installation. I have used `-cert-manager-as-subchart` to generate cert-manager dependency. Once I have tried installing the chart, I got an error:

```
Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: [resource mapping not found for name: "OPERATOR_NAME-serving-cert" namespace: "default" from "": no matches for kind "Certificate" in version "cert-manager.io/v1"
ensure CRDs are installed first, resource mapping not found for name: "<OPERATOR_NAME>-selfsigned-issuer" namespace: "default" from "": no matches for kind "Issuer" in version "cert-manager.io/v1"
ensure CRDs are installed first]
```
It seems that it's a frequent issue across many operators([GitLab operator example](https://gitlab.com/gitlab-org/cloud-native/gitlab-operator/-/issues/1268)) due to missing cert-manager CRDs. One of the solutions is to install CRDs prior to the chart installation and setting `--set cert-manager.installCRD` to `false`.
 
I see that that the value for `cert-manager.installCRD` is hardcoded to `true` [here](https://github.com/arttor/helmify/blob/1a32d2058e682dc9a2486615c5a6b5ebf71bab05/pkg/helm/chart.go#L111C19-L111C22). I would like to contribute to the project and create another flag to change value of this field. 

Let me know what do you think.

**Comment by arttor** [comment]
yes, it totally makes sense to add CLI option to set default value. Just make sure that default value remain the same.
